src/AutoRest.CSharp/Common/Output/Models/RestClientBuilder.cs

```diff
@@ -455,6 +457,12 @@ private static RequestParameterSerializationStyle GetSerializationStyle(RequestP
             }
         }
 
+        private static bool GetExplode(RequestParameter requestParameter)
+        {
+            var httpParameter = requestParameter.Protocol.Http as HttpParameter;
+            return httpParameter?.Explode ?? false;
```

Comment:
`requestParameter.Protocol.Http is HttpParameter httpParameter && httpParameter.Explode == true`